frb_codegen/src/commands.rs

```diff
@@ -274,3 +283,88 @@ pub fn build_runner(dart_root: &str) -> Result {
     }
     Ok(())
 }
+
+pub fn guess_context(dart_root: &str) -> std::result::Result<Context, Error> {
+    info!("Guessing context the runner is run into");
```

Comment:
uber-nit: Well maybe `debug!` is enough

Reply:
oh yes indeed, in 1aa61fa9336101263bfe2ec47f8055986dcb4429